tests/mcp/test_server.py

```
@@ -0,0 +1,112 @@
+        (tmp_path / "hunts").mkdir()
+        (tmp_path / "research").mkdir()
+        (tmp_path / "investigations").mkdir()
+
+        from athf.mcp.server import create_server
+
+        server = create_server(str(tmp_path))
+
+        # Get registered tool names
+        import asyncio
+
+        async def get_tools():
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `get_tools`

(ANN202)